docs/installation/install-client.rst

```diff
@@ -119,38 +119,29 @@ Whenever you need to run Flocker CLI commands, ensure you are in the virtualenv:
 Installing on OS X
 ==================
 
-Install the `Homebrew`_ package manager.
-
-Make sure Homebrew has no issues:
-
-.. prompt:: bash alice@mercury:~$
-
-   brew doctor
-
-Fix anything which ``brew doctor`` recommends that you fix by following the instructions it outputs.
-
-If you have a previous version of Flocker tapped, you can run the following to remove it:
+Install ``pip`` and ``virtualenv`` on your machine at the system level:
 
 .. prompt:: bash alice@mercury:~$
 
-   brew uninstall flocker-<old version>
+   sudo python -m ensurepip
+   sudo pip install virtualenv
 
-Add the ``ClusterHQ/tap`` tap to Homebrew and install ``flocker``:
+To install ``flocker-cli`` in a Python virtualenv, run:
```

Comment:
:question: I think it's confusing to highlight ``flocker-cli`` this way, since there's no such tool or package, but I see that's how it's formatted elsewhere....so fine. Might be better if we talked about installing ``flocker-ca`` specifically since that's the only client command of interest these days....until we have a proper ``flocker-volumes`` tool....and even then perhaps we should make it a ``flocker volumes`` and a ``flocker ca`` style subcommand.